packages/js/src/ui/novuUI.tsx

```diff
@@ -1,12 +1,13 @@
 import { MountableElement, render } from 'solid-js/web';
 import type { NovuOptions } from '../novu';
 import { Appearance } from './context';
-import './index.css';
 import { generateRandomString } from './helpers';
 import { Localization } from './context/LocalizationContext';
-import { NovuComponent, NovuComponentControls, NovuComponentName, Renderer } from './components/Renderer';
+import { NovuComponent, NovuComponentName, Renderer } from './components/Renderer';
 import { NovuProviderProps } from './types';
 import { createSignal } from 'solid-js';
+//@ts-expect-error inline import esbuild syntax
+import css from 'directcss:./index.directcss';
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Not a fan of this hacky approach. But it's fine for now. We should revisit this later for a better solution.
‏

Reply:
We either will use a CDN for this or find another solution altogether. This is to unblock us for now.